packages/docs/package.json

```diff
@@ -7,6 +7,7 @@
     "@alteryx/icons": "^0.0.1",
     "@alteryx/react-comms": "^0.0.1",
     "@alteryx/ui": "^0.0.3",
+    "@material-ui/core": "^4.11.4",
```

Comment:
Was this added for ts definitions?

Reply:
Yeah, the docs builds failed looking for references to @material-ui/core definitions